deploy/vagrant/bin/run_docker.sh

```diff
@@ -23,7 +23,7 @@ vagrant up --provider=docker --no-provision
 HOSTS=`vagrant ssh-config 2>/dev/null |grep -w Host |awk '{print $2}'`
 for h in $HOSTS; do
  addr=`vagrant ssh $h -c "ifconfig eth0" |grep -w inet|awk '{print $2}'`
- echo ${addr} ${h}".local" >> files/hosts
```

Comment:
Originally, it works. But after my modification of deploy module, the two platforms are not tested. When the module is quite stable(after this PR), I will test the two platforms.